pkgs/by-name/gn/gnome-keyring/package.nix

```diff
@@ -64,13 +62,8 @@ stdenv.mkDerivation rec {
   configureFlags = [
     "--with-pkcs11-config=${placeholder "out"}/etc/pkcs11/" # installation directories
     "--with-pkcs11-modules=${placeholder "out"}/lib/pkcs11/"
-    # gnome-keyring doesn't build with ssh-agent by default anymore, we need to
-    # switch to using gcr https://github.com/NixOS/nixpkgs/issues/140824
-    "--enable-ssh-agent"
```

Comment:
I noticed that `$out/etc/xdg/autostart/gnome-keyring-ssh.desktop` is still around, and causing the env var to be set still. It might be necessary to remove it in `postFixup` to really disable everything.